docs/core/tools/dotnet-sln.md

```diff
@@ -35,42 +34,64 @@ dotnet new sln
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
 
 ## Options
 
 - **`-h|--help`**
 
-  Prints out a short help for the command.
+  Prints options for the command.
 
 ## Commands
 
+### `list`
+
+Lists all projects in a solution file.
+
+#### Synopsis
+
+```dotnetcli
+dotnet sln list [-h|--help]
+```
+  
+#### Arguments
+
+- **`SOLUTION_FILE`**
+
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
+
+#### Options
+
+- **`-h|--help`**
+
+  Prints out options for the command.
+  
 ### `add`
 
-Adds a project or multiple projects to the solution file.
+Adds one or more projects to the solution file.
 
 #### Synopsis
 
 ```dotnetcli
-dotnet sln [<SOLUTION_FILE>] add [--in-root] [-s|--solution-folder] <PROJECT_PATH>
+dotnet sln [<SOLUTION_FILE>] add [--in-root] [-s|--solution-folder <SOLUTION_FOLDER>] <PROJECT_PATH> [<PROJECT_PATH>...]
 dotnet sln add [-h|--help]
 ```
 
 #### Arguments
 
 - **`SOLUTION_FILE`**
 
-  The solution file to use. If not specified, the command searches the current directory for one. If there are multiple solution files in the directory, one must be specified.
+  The solution file to use. The command searches the current directory for one if it's left unspecified, failing if there are multiple solution files.
 
 - **`PROJECT_PATH`**
 
-  The path to the project to add to the solution. Add multiple projects by adding one after the other separated by spaces. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
+  The path to the project or projects to add to the solution. Unix/Linux shell [globbing pattern](https://en.wikipedia.org/wiki/Glob_(programming)) expansions are processed correctly by the `dotnet sln` command.
 
 #### Options
 
 - **`-h|--help`**
 
-  Prints out a short help for the command.
+  Prints out a short list of options for the command.
```

Comment:
this is description for all --help options across CLI docs. If this is preferred version, we'd have to change across all docs.